contextproject-ms3/src/main/java/contextproject/loaders/PlaylistLoader.java

```diff
@@ -0,0 +1,36 @@
+package contextproject.loaders;
+
+import java.io.File;
+import java.util.ArrayList;
+
+/**
+ * Interface for the loaders to ensure it has the right functionality.
+ *
+ */
+public interface PlaylistLoader {
+
+  /**
+   * Method to get a list of all the files that are in the folder.
+   * 
+   * @return a list.
+   */
+  public ArrayList<String> getList();
```

Comment:
The purpose of a PlaylistLoader should be to load everything into a Playlist once, so I think the interface can be a lot easier. Maybe just a ```public ArrayList<String> load();```? (and remove everything else)